### PR TITLE
Script to migrate cluster from OpenShiftSDN to OVNKubernetes

### PR DESCRIPTION
**Pull request by mukrishn**
### Description

- Steps are documented here - https://docs.google.com/document/d/1KrQsFpHIuFg5Lv03muY__0-MlwftMFqgEsiroCkrBA0/edit#heading=h.24easisenadj 
- Tested on ROSA cluster

### CODE REVIEW

File: workloads/sdn2ovn/run.sh [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+set -x
+
+source env.sh
+
+# _wait_for <resource> <resource_name> <desired_state> <timeout in minutes>
+_wait_for(){
+    echo "Waiting for $2 $1 to be $3 in $4 Minutes"
+    oc wait --for=condition=$3 --timeout=$4m $1 $2
+}
+_reboot(){
```
Comment (smalleni) [inline workloads/sdn2ovn/run.sh:11]: As per recommendations you should reboot using AWS console, not by drain and cordoning.

Comment (smalleni) [reply workloads/sdn2ovn/run.sh:11]: Also cordon should be before drain anyway, no?

Comment (smalleni) [reply workloads/sdn2ovn/run.sh:11]: I'm checking if the cordon/drain procedure is even needed

Comment (mukrishn) [reply workloads/sdn2ovn/run.sh:11]: Is it okay to reboot nodes using debug command ? 

`oc debug  node/<NODE> -- chroot /host shutdown -r -t 3`

Comment (smalleni) [reply workloads/sdn2ovn/run.sh:11]: Yes that should be good.

**Comment by mukrishn** [comment]
@venkataanil FYI

**Review by venkataanil — APPROVED** [review]
This script is very helpful and  I was able to migrate with the steps mentioned here. Please merge it.

**Review by krishvoor — APPROVED** [review]
lgtm

**Comment by krishvoor** [comment]
@mukrishn can you plz rebase this?
@smalleni: we can leverage this for https://issues.redhat.com/browse/PERFSCALE-2349

**Review by venkataanil — APPROVED** [review]
LGTM

**Review by krishvoor — APPROVED** [review]
lgtm

**Comment by mukrishn** [comment]
Please Approve this to merge.

**Review by afcollins — APPROVED** [review]
LGTM

**Pull request merged**